targets/ipq40xx/profiles/default/profile_images

```diff
@@ -0,0 +1,19 @@
+_jalapeno-squashfs-
```

Comment:
Will accept as is, but in a subsequent PR can you please add the manufacturer string to the front of this? For consistency more than anything else. Thanks!

Reply:
Just when you get time or are sending another round of changes. Not urgent!